src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -103,9 +107,28 @@ struct CPUStreamsExecutor::Impl {
                                 return p.second > streamId_wrapped;
                             })
                             ->first;
-                    _taskArena.reset(new custom::task_arena{custom::task_arena::constraints{}
-                                                                .set_core_type(selected_core_type)
-                                                                .set_max_concurrency(concurrency)});
+                    const auto max_concurrency = selected_core_type == 0 ? _impl->_config._threads_per_stream_small
+                                                                         : _impl->_config._threads_per_stream_big;
+                    const auto stream_id =
+                        selected_core_type == 0 ? _streamId - _impl->_config._big_core_streams : _streamId;
+                    const auto thread_binding_step = selected_core_type == 0 ? _impl->_config._threadBindingStep : 2;
+                    const auto cpu_idx_offset = selected_core_type == 0 ? _impl->_config._small_core_offset : 1;
```

Comment:
for me this is not clear why cpu_idx_offset for big cores is 1, not 0. maybe some comment is required

Reply:
Reuse logic from "CORES" affinity.